### PR TITLE
Remove terminal specific command from e2e test file

### PR DESCRIPTION
**Pull request by amitkrout**
## What is the purpose of this change? What does it change?

Update the test file to run it on all oc supported platform including windows
## Was the change discussed in an issue?
fixes - Part of #1129

## How to test changes?

make test-main-e2e

### CODE REVIEW

**Comment by amitkrout** [comment]
Yet to update, So WIP
```
// get the name of the pod
				podName := runCmdShouldPass("oc get pods | grep nodejs | awk '{print $1}' | tr -d '\n'")
```

**Comment by amitkrout** [comment]
Locally i am unable to reproduce the reported ```ci/circleci```. Might be a ```circleci``` glitch. Re triggering the job

**Comment by amitkrout** [comment]
Rebased

File: tests/e2e/utils.go [hunk 1]
```diff
@@ -99,6 +100,52 @@ func getActiveApplication() string {
 	return strings.TrimSpace(odoActiveApp)
 }
 
+// returns a local config value of given key and
+// returns an empty string if value is not set
+func findLocalConfigValueOfGivenKey(key string) string {
```
Comment (kadel) [inline tests/e2e/utils.go]: this is an unnecessary long function name. Why not just `getConfigValue(key string)`?

File: tests/e2e/utils.go [hunk 2]
```diff
@@ -99,6 +100,52 @@ func getActiveApplication() string {
...
+	trimKeyValue := strings.TrimSpace(odoConfigKeyValue)
+	if strings.Compare(key, trimKeyValue) != 0 {
+		return strings.TrimSpace(strings.SplitN(trimKeyValue, " ", 2)[1])
+	}
+	return ""
+}
+
+// returns a global config value of given key and
+// returns an empty string if value is not set
+func findGlobalConfigValueOfGivenKey(key string) string {
```
Comment (kadel) [inline tests/e2e/utils.go]: it is not called gloabl config, it should be preferences. How about just `getPreferenceValue`?

File: tests/e2e/utils.go (hunk 1 above)
Comment (amitkrout) [reply tests/e2e/utils.go]: I like the suggested name :)

Comment (amitkrout) [reply tests/e2e/utils.go]: Done

File: tests/e2e/utils.go (hunk 2 above)
Comment (amitkrout) [reply tests/e2e/utils.go]: Done

File: tests/e2e/utils.go [hunk 3]
```diff
@@ -99,6 +100,52 @@ func getActiveApplication() string {
 	return strings.TrimSpace(odoActiveApp)
 }
 
+// returns a local config value of given key and
```
Comment (syamgk) [inline tests/e2e/utils.go]: given key ~~and~~ or

File: tests/e2e/utils.go [hunk 4]
```diff
@@ -99,6 +100,52 @@ func getActiveApplication() string {
...
+		return fmt.Sprintf("%s not found", key)
+	}
+	trimKeyValue := strings.TrimSpace(odoConfigKeyValue)
+	if strings.Compare(key, trimKeyValue) != 0 {
+		return strings.TrimSpace(strings.SplitN(trimKeyValue, " ", 2)[1])
+	}
+	return ""
+}
+
+// returns a global config value of given key and
```
Comment (syamgk) [inline tests/e2e/utils.go]: given key ~~and~~ or

File: tests/e2e/utils.go (hunk 3 above)
Comment (amitkrout) [reply tests/e2e/utils.go]: Done

File: tests/e2e/utils.go (hunk 4 above)
Comment (amitkrout) [reply tests/e2e/utils.go]: Done

**Pull request merged**